avr-hal-generic/src/wdt.rs

```diff
@@ -55,6 +61,21 @@ pub trait WdtOps<H> {
     fn raw_stop(&mut self);
 }
 
+#[derive(Copy, Clone)]
+/// The mode that dictates how the watchdog should behave.
+/// This needs to be called **after** the `start` function.
```

Comment:
You cannot "call" an enum.  This statement probably belongs into the documentation of one of the methods?